### ISSUE TITLE
How do I start miner?

### ISSUE THREAD

**Issue by s60912frank** [dexon-foundation/dexon#27]
#### System information

Geth version: `gdex 1.8.17-stable`
OS & Version:  `Alpine (Docker container)`

#### Expected behaviour
Miner start through miner.start() in console, or gdex with --mine option

#### Actual behaviour
'Miner' is not defined

I saw there is a module named "dexcon" in gdex console, but calling "dexcon" is not defined.

**Comment by aitjcize** [comment]
Hi,

Thanks for the interest in running a `miner`. We actually call it a `BlockProposer` instead of a `miner`. There is an option `--bp` to enable block proposer mode. However, to run a BP, you need to stake your public key in the governance contract. Currently, we have not opened the peer network to outside yet, so it's impossible to run a BP even if you have enough stake.

We will release the instruction and setup to run a BP very soon (ETA ~ 1.5 weeks) from now. Please stay tunned.

**Comment by s60912frank** [comment]
Thank you for your answer! Now I can start a node and it tell me I cannot vote (which is obviously I didn't stake).
But another question, is it possible to build a private chain now? Now I set few accounts to be pre-fund, and stake in genesis block, but I got `panic: non-positive interval for NewTicker`, which I think there may be a option to set blocktime in genesis block, is this listed in somewhere?
Thanks.
